vendorconsent/consent_test.go

```diff
@@ -3,6 +3,8 @@ package vendorconsent
 import (
 	"encoding/base64"
 	"testing"
+
+	tcf1 "github.com/prebid/go-gdpr/vendorconsent/tcf1"
 )
 
 func TestIsSet(t *testing.T) {
```

Comment:
Folders `vendorconsent/`, `vendorconsent/tcf1/`, and `vendorconsent/tcf2/` all define their own, identical `isSet` function. Maybe we can put only one definition of it in `vendorconsent/utils` object in a future PR.